source/adapters/cuda/platform.hpp

```diff
@@ -9,7 +9,8 @@
 //===----------------------------------------------------------------------===//
 #pragma once
 
-#include <ur/ur.hpp>
+#include <memory>
+#include <ur_api.h>
```

Comment:
Are the headers not ordered hierarchically e.g local, 3rd-party, standard-library or similar?